ConnectorArcGIS/Bindings/BasicConnectorBinding.cs

```diff
@@ -23,7 +23,8 @@ public BasicConnectorBinding(ArcGisDocumentStore store)
 
   public string GetSourceApplicationVersion() => "3";
 
-  public string GetConnectorVersion() => Assembly.GetAssembly(GetType()).GetNameAndVersion().Version;
+  public string GetConnectorVersion() =>
+    typeof(BasicConnectorBinding).Assembly.GetNameAndVersion().Version ?? "No version";
```

Comment:
This is odd. Even development versions of our connectors should have a version set, not having a version.

Was this failing for some reason?

Reply:
This was a copy from Rhino.

The `version` property is nullable, so we needed to resolve the warning, so I copied what rhino was doing. I would prefer to throw here instead if we expect version to always be set.

Reply:
Fine, I accept for now :)